src/Uno.UI.Runtime.Skia/GLCanvasElement.GLVisual.cs

```diff
@@ -0,0 +1,35 @@
+using Microsoft.UI.Composition;
+using SkiaSharp;
+using Uno.UI.Composition;
+
+namespace Microsoft.UI.Xaml.Controls;
+
+public abstract partial class GLCanvasElement
+{
+	private class GLVisual(GLCanvasElement owner, Compositor compositor) : Visual(compositor)
```

Comment:
Not related to transparency, but resizing the window is giving me this kind of placement:

<img width="467" alt="image" src="https://github.com/user-attachments/assets/952d5127-245d-4e92-af18-debe922cc39b">

Reply:
This should have been fixed.